consumers/writers/postgres/consumer.go

```diff
@@ -51,7 +51,7 @@ func (pr postgresRepo) Consume(message interface{}) (err error) {
 	}
 }
 
-func (pr postgresRepo) saveSenml(messages interface{}) error {
+func (pr postgresRepo) saveSenml(messages interface{}) (err error) {
```

Comment:
Why do we need this named param, when `err` is not used in the function?

Reply:
Because the deferred function is using `err` var. Before we named it, the deferred function was using the wrong error value - the one from the `BeginTxx` method, rather than the one returned from `saveSenml`.